src/redux/app/reducer.ts

```diff
@@ -15,10 +17,14 @@ const intialState: IAppState = {
     testNet: true,
     networks: {
         [Blockchain.ETHEREUM]: {
```

Comment:
we have to decide if the ordering affects all wallets or we have ordering for each wallet separatly

Reply:
also we should add a new property `blockchains` on the same level with `networks`. `networks` should have only networks info.